scripts/setup/install.ps1

```diff
@@ -42,6 +42,48 @@ Param(
 
 $_rollbackStore = @{}
 
+function Retry-Command
+{
+    param (
+    [Parameter(Mandatory=$true)][string]$command,
+    [Parameter(Mandatory=$true)][hashtable]$args,
+    [Parameter(Mandatory=$false)][int]$retries = 2,
+    [Parameter(Mandatory=$false)][int]$secondsDelay = 1,
```

Comment:
1 second delay seems a bit short, should we increase number of count or increase delay.

Reply:
Let's keep this for this now. For I confirmed this fixed the issue. If the issue happens again, we can increase this or we will need to figure out the root cause of the failure.